app/api/routes/user.py

```diff
@@ -1,8 +1,29 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, HTTPException, status
+from models import UserCreateRequestModel, User  
 
 router = APIRouter()
 
-# make sure to remove this function
-@router.get("/example")
-def example():
-    return {"message": "this is an example"}
+
+users_db = {}
+
+
+@router.post("/users/", response_model=User, status_code=status.HTTP_201_CREATED)
```

Comment:
Why was the UserCreateResponseModel removed? Password shouldn't be part of the response, and the same goes for the update date.

Reply:
I fixed it, sorry i didn't realize it then.

Reply:
UserCreateResponseModel should include the creation date. Please double-check the task to ensure the correct response format.

Reply:
yes iam just missing it, it's done.